web/admin/job.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Community.ExAdmin.Job do
+  use ExAdmin.Register
+
+  register_resource Community.Job do
+
+  end
```

Comment:
Yes. I don't need the empty line though.

Without the call at all, the resource is not registered. It does not take the option to call the `reigster_resource` without the `do/end`.

Thanks for loooing out!